tests/framework/installer/ceph_manifests.go

```diff
@@ -419,6 +422,9 @@ metadata:
   name: ` + settings.Namespace + `
   namespace: ` + settings.Namespace + `
 spec:
+  cephVersion:
```

Comment:
Note: As we still have this file with the yamls, it does not seem that the issue I mentioned earlier is fixed by this.

Reply:
These yamls are the intended approach for testing. I'm still not sure the issue you're referring to.